modules/ocf/manifests/serial_getty.pp

```diff
@@ -0,0 +1,8 @@
+class ocf::serial_getty {
+  service { 'serial-getty@ttyS0':
+    ensure   => running,
```

Comment:
I don't have a good alternative if we care that it gets started immediately, unfortunately. On stretch you could do

```puppet
exec { 'systemctl enable --now serial-getty@ttyS0':
  creates => '/etc/systemd/system/getty.target.wants/serial-getty@ttyS0.service',
}
```

but jessie doesn't support `--now`